src/Carnac.Logic/KeyProvider.cs

```diff
@@ -105,7 +102,7 @@ KeyPress ToCarnacKeyPress(InterceptKeyEventArgs interceptKeyEventArgs)
             var isLetter = interceptKeyEventArgs.Key >= Keys.A &&
                            interceptKeyEventArgs.Key <= Keys.Z;
 
-            var inputs = ToInputs(isLetter, winKeyPressed, interceptKeyEventArgs);
+            var inputs = ToInputs(isLetter, winKeyPressed, interceptKeyEventArgs).ToArray();
```

Comment:
Given this is internal, would it be worth just changing the return type on `ToInputs` to make it clear this is necessary?